README.md

```diff
@@ -1,14 +1,77 @@
 # webpack-partial
 
-Intelligently merge webpack configuration files.
+Intelligently compose [webpack] configuration files.
 
 ![build status](http://img.shields.io/travis/webpack-config/webpack-partial/master.svg?style=flat)
 ![coverage](http://img.shields.io/coveralls/webpack-config/webpack-partial/master.svg?style=flat)
 ![license](http://img.shields.io/npm/l/webpack-partial.svg?style=flat)
 ![version](http://img.shields.io/npm/v/webpack-partial.svg?style=flat)
 ![downloads](http://img.shields.io/npm/dm/webpack-partial.svg?style=flat)
 
-Take a multiple webpack configurations (or functions generating webpack configurations) and merge them. Usually used in the manner of `partial(fullConfig, partialConfig)` to return a new complete configuration.
+
+## Usage
+
+```sh
+npm install --save webpack-partial
+```
+
+Making [webpack] configurations composable is tricky. So we provide several helpers to ease the composition process. Each helper is of the form `(arg1, arg2, ..., webpackConfig)`. All functions are curried and the last argument is always a webpack configuration object allowing you to chain helpers together easily with `compose`.
+
+Simple single-plugin example:
+
+```javascript
+import plugin from 'webpack-partial/plugin';
+import StatsWebpackPlugin from 'stats-webpack-plugin';
+
+const config = plugin(new StatsWebpackPlugin(), {/* webpack config here */});
+```
+
+Now with multiple plugins:
+
+```javascript
+import plugin from 'webpack-partial/plugin';
+import StatsWebpackPlugin from 'stats-webpack-plugin';
+import ExtractTextWebpackPlugin from 'extract-text-webpack-plugin';
+
+const config = plugin(
+  new StatsWebpackPlugin(),
+  plugin(
+    new ExtractTextWebpackPlugin(),
+    /* webpack config here */
+  )
+);
+```
+
+And using composition:
+
+```javascript
+import compose from 'lodash/fp/compose';
```

Comment:
I've stopped using `compose` in favor of `flow` for a couple of reasons. First and less important is that compose is relegated to an alias in lodash and I'd prefer to use full methods since it's easier to point people to docs. Also, JDD uses the word compose all over the place so searching for the docs for the actual compose method is a pain in the butt. More important is that it is an alias for `flowRight` which can really mess up people's mental model if they're not use to the right -> left dataflow.